### DISCUSSION TITLE
Eleventy and Apple M1-based Macs

### DISCUSSION THREAD

**Discussion by rtkadalusong** [11ty/eleventy#2185] [Q&A] [answered]
Hello team,

I'm wondering if there are any issues with running Eleventy on an Apple M1-based machine. From using Google-fu, it looks like Javascript has no issues whatsoever. My biggest question is regarding Liquid, as it is based on Ruby. Some users had minor issues installing Ruby (and Jekyll) into their M1-based machines and I wonder if this issue trickles down to Eleventy.

Any insight is appreciated, thanks!

**Comment by cfjedimaster** [comment, marked as answer]
Liquid is written in JavaScript. Jekyll is Ruby, but Liquid itself is JS - https://liquidjs.com/.

**Reply by rtkadalusong** [reply]
That clears things up. Thanks!

**Reply by pdehaan** [reply]
FWIW, I am running Eleventy 1.0 and Eleventy 0.12 on an M1 Max rMBP [and older M1 Mac mini] with zero issues w/ Node/npm or nvm.

**Reply by zachleat** [reply]
fwiw I have an M1 Macbook Air 2020
